.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,26 @@
+---
+name: Bug report
+about: Create a report to help us improve
+
```

Comment:
Can we add default labels here - `labels: bug, triage me` ?

Reply:
Sure, done. I made the triage label be `triage-me` since it seemed like labels in the `opencensus-node` repo had dashes rather than spaces.